core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1948,6 +1956,20 @@ private int toColumnType(GenericLexer lexer, CharSequence tok) throws SqlExcepti
         return tok;
     }
 
+    private SqlException duplicateColumnException(int position, CharSequence colName) {
+        return duplicateColumnException(position, colName, null);
+    }
+
+    private SqlException duplicateColumnException(int position, CharSequence colName, CharSequence additionalMessage) {
```

Comment:
I'd move this method as a static to SqlException with usage of `Misc.getThreadLocalBuilder()` and use it from there all around the code

Reply:
the three suggestions have been incorporated